core/lib/spree/localized_number.rb

```diff
@@ -1,21 +1,29 @@
 module Spree
   class LocalizedNumber
 
-    # Strips all non-price-like characters from the number, taking into account locale settings.
-    def self.parse(number)
-      return number unless number.is_a?(String)
+    class << self
+      # Strips all non-price-like characters from the number, taking into account locale settings.
+      def parse!(number, default = nil)
```

Comment:
Why would you like to put `!` in this method's name? It does not look particularly dangerous :)